usaspending_api/api_contracts/contracts/v2/idvs/activity.md

```diff
@@ -46,6 +46,7 @@ List child and grandchild awards for a given IDV
                         "award_id": 1867804,
                         "awarding_agency": "Department of Veterans Affairs",
                         "awarding_agency_id": 561,
+                        "awarding_agency_slug": "department-of-veterans-affairs,
```

Comment:
This needs a final quote, right?

Reply:
Good catch! Yeh @collinwr I would add those so we don't forget the we can get this merged.